src/dotless.Core/Parser/Tree/Rule.cs

```diff
@@ -14,6 +16,12 @@ public class Rule : Node
         public NodeList PostNameComments { get; set; }
         public bool IsSemiColonRequired { get; set; }
         public bool Variadic { get; set; }
+        public string Merge { get; set; }
+
+        public bool Important
+        {
+            get { return Value is Value && ((Value) Value).Important == "!important"; }
```

Comment:
Will do, this is the first place I got a bit uncertain if I could use c#6 features or not ;)